### DISCUSSION TITLE
Unbalanced celery jobs distribution

### DISCUSSION THREAD

**Discussion by XhivaW** [apache/airflow#35832] [Q&A]
Hi all, 
I meet a problem with Airflow 1.10.12 about unbalanced celery jobs as pic shows
<img width="754" alt="image" src="https://github.com/apache/airflow/assets/12463092/cd578f84-c6bc-460b-b000-d10ce2a67adf">
just wonder how the airflow send jobs to workers and if there's way to adjust
Thx in advance

**Comment by Taragolis** [comment]
Difficult to say Airflow 1.10.x [EOL](https://airflow.apache.org/docs/apache-airflow/stable/installation/supported-versions.html) for more than 2 years, not many people use it in production these days. Many changes have been made in [Airflow](https://airflow.apache.org/docs/apache-airflow/stable/release_notes.html) as well in [Celery](https://docs.celeryq.dev/en/stable/changelog.html) since then

**Comment by potiuk** [comment]
Yeah. You might hope one of the few remaining users of Airflow 1.10 and probably far less of them using 1.10.12 that has been released 3.5 years ago and then reached end of life more than 2 years ago. But I am afraid you are in this position:

![image](https://github.com/apache/airflow/assets/595491/d7f03760-8588-4269-9176-5a379b6fadaf)


But seriously. In case you have not realised.

 You are running seriously outdated software that has not received any fix for more than 3 years and stopped receiving even security fixes more than 2 years ago. As of more or less 1 year we mostly respond to questions about 1.10.* with this XKCD comic above, because almost no-one who worked when that End-of-life software has been working on Airflow project back when it was created and even if they did, they don't remember anything from those days and even if you want to pay for support probably there is no-one who would want to look back 2 years past the moment when the life of the product has ended.

Just a friendly advice Maybe consider upgrading? Unless you want to be like the comic hero above.
